patches/minecraft/net/minecraft/entity/LivingEntity.java.patch

```diff
@@ -195,9 +195,12 @@
     }
  
     protected void func_213337_cE() {
-@@ -1240,6 +_,7 @@
+@@ -1238,8 +_,9 @@
+ 
+    protected void func_226294_cV_() {
        if (!this.field_70170_p.field_72995_K && (this.func_70684_aJ() || this.field_70718_bc > 0 && this.func_146066_aG() && this.field_70170_p.func_82736_K().func_223586_b(GameRules.field_223602_e))) {
-          int i = this.func_70693_a(this.field_70717_bb);
+-         int i = this.func_70693_a(this.field_70717_bb);
++         int i = func_70693_a(this.field_70717_bb);
```

Comment:
This patch change is a bit redundant now, its just removing the `this`

Reply:
I have readded 'this' to match the previous patch.